rust-bins/src/bin/pg_test_tool.rs

```diff
@@ -0,0 +1,465 @@
+//! # Privacy Guardian Test Tool
+//!
+//! An application to test the functionality of privacy guardian keys.
+use clap::AppSettings;
+use client_server_helpers::*;
+use concordium_base::{
+    common::*,
+    curve_arithmetic::{Curve, Value},
+    elgamal::{
+        decrypt_from_chunks_given_generator, encrypt_in_chunks_given_generator, Cipher, PublicKey,
+    },
+    id::{constants::ArCurve, types::*},
+};
+use dialoguer::Input;
+use hex::encode;
+use rand::{rngs::ThreadRng, seq::IteratorRandom, thread_rng};
+use sha2::{Digest, Sha256};
+use std::{
+    fmt::Debug,
+    path::{Path, PathBuf},
+    str::from_utf8,
+};
+use structopt::StructOpt;
+
+/// The path to the list of BIP39 words.
+const BIP39_ENGLISH: &str = include_str!("data/BIP39English.txt");
+
+/// List of BIP39 words. There is a test that checks that this list has correct
+/// length, so there is no need to check when using this in the tool.
+fn bip39_words() -> impl Iterator<Item = &'static str> { BIP39_ENGLISH.split_whitespace() }
```

Comment:
This looks like copy-paste from lib.rs. I would suggest to use the function from lib.rs instead 😊